### PR TITLE
tests: add proxy test for a system without internet

### PR DESCRIPTION
**Pull request by renanrodrigo**
## Desired commit type
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: features/proxy_config.feature [hunk 1]
```diff
@@ -1183,3 +1183,34 @@ Feature: Proxy configuration
            | bionic  |
            | focal   |
            | jammy   |
+
+    @slow
+    @series.jammy
+    @uses.config.machine_type.lxd.vm
+    Scenario: Enable realtime kernel through proxy on a machine with no internet
+        Given a `jammy` machine with ubuntu-advantage-tools installed
+        When I disable any internet connection on the machine
```
Comment (lucasmoura) [inline features/proxy_config.feature:1192]: That's really great :rocket: 
But I think we can add that step to the other releases as well right ?

Comment (renanrodrigo) [reply features/proxy_config.feature:1192]: I used this one here to test the realtime-kernel enablement (which implies Jammy).
If you think we can have other tests using those features, then yes, we can add more
Just let me know which ones should be in this PR

Comment (orndorffgrant) [reply features/proxy_config.feature:1192]: I think this is good for now, we can expand later if needed.

Comment (lucasmoura) [reply features/proxy_config.feature:1192]: I think you are right, we could add other similar tests for the older releases, but maybe it is not required

**Review by lucasmoura — APPROVED** [review]

**Review by orndorffgrant — APPROVED** [review]

**Pull request merged**